lib/mfem/SampleMesh.cpp

```diff
@@ -377,12 +393,14 @@ void BuildSampleMesh(ParMesh& pmesh, vector<ParFiniteElementSpace*> & fespace,
         }
 
         pmesh.GetElementVertices(elId, elVert);
-        MFEM_VERIFY(numElVert == elVert.Size(), "");  // Assuming a uniform element type in the mesh.
+        MFEM_VERIFY(numElVert == elVert.Size(),
+                    "");  // Assuming a uniform element type in the mesh.
```

Comment:
The comment could go in the string that is the second argument of `MFEM_VERIFY` (yes, I wrote this).